packages/actor-rdf-dereference-file/components/Actor/RdfDereference/File.jsonld

```diff
@@ -0,0 +1,64 @@
+{
+  "@context": [
+    "https://linkedsoftwaredependencies.org/contexts/comunica-actor-rdf-dereference-file.jsonld",
+    "https://linkedsoftwaredependencies.org/contexts/comunica-bus-rdf-dereference.jsonld"
+  ],
+  "@id": "npmd:@comunica/actor-rdf-dereference-file",
+  "components": [
+    {
+      "@id": "cardf:Actor/RdfDereference/File",
+      "@type": "Class",
+      "extends": "cbrd:Actor/RdfDereference",
+      "requireElement": "ActorRdfDereferenceFile",
+      "comment": "A comunica File RDF Dereference Actor.",
+      "parameters": [
+        {
+          "@id": "cardf:mediatorRdfParse",
+          "comment": "The RDF parse mediator",
+          "required": true,
+          "unique": true
+        },
+        {
+          "@id": "cardf:mediaMappingEntry",
+          "comment": "Maps an extension to a file type",
+          "required": true,
+          "default": [
+            { "cardf:extension": ".ttl",      "cardf:mediaType": "text/turtle" },
+            { "cardf:extension": ".turtle",   "cardf:mediaType": "text/turtle" },
+            { "cardf:extension": ".nt",       "cardf:mediaType": "text/turtle" },
```

Comment:
Media type of this and all below are wrong. (I know that N3 can handle all of them anyways, but this is not necessarily like this for all possible parsers)